src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard.tsx

```diff
@@ -160,22 +174,89 @@ export default class SentryApplicationDashboard extends AsyncView<Props, State>
     );
   }
 
+  renderIntegrationViews() {
+    const {views} = this.state.interactions;
+    return (
+      <Panel>
+        <PanelHeader>{t('Integration Views')}</PanelHeader>
+
+        <InteractionsChart data={{Views: views}} />
+      </Panel>
+    );
+  }
+
+  renderComponentInteractions() {
+    const {component_interactions} = this.state.interactions;
+
+    return (
+      <Panel>
+        <PanelHeader>{t('Component Interactions')}</PanelHeader>
+
+        <PanelBody>
+          <InteractionsChart data={component_interactions} />
+        </PanelBody>
+
+        <PanelFooter>
+          <StyledFooter>
+            <strong>{t('stacktrace-link:')}</strong>{' '}
+            {t('Each click on the link counts as one interaction')}
+            <br />
+            <strong>{t('issue-link:')}</strong>{' '}
+            {t('Each open of the issue link modal counts as one interaction')}
+          </StyledFooter>
+        </PanelFooter>
+      </Panel>
+    );
+  }
+
   renderBody() {
-    const {app, loading} = this.state;
-    if (!loading && app && app.status !== 'published') {
-      return <LoadingError />;
-    }
+    const {app} = this.state;
 
     return (
       <div>
         {app && <SettingsPageHeader title={app.name} />}
-        {this.renderInstallData()}
+        {app && app.status === 'published' && this.renderInstallData()}
         {this.renderErrorLog()}
+        {app && app.status === 'published' && this.renderIntegrationViews()}
```

Comment:
@nolachen I think there could be value here in seeing the interactions even if the app is not published.

Reply:
@scefali this will just restrict the Integration Views. The Component Interactions will still be shown for internal/unpublished apps. Do you think the Integration Views would be valuable for internal/unpublished?

Reply:
@nolachen Whoops..yea I am confused. What you have is fine.